Makefile

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
Is this makefile part of the boiler plate generation? It's good stuff, but I'm worried it also teaches the developer to not have to actually understand what's going on when working with this app.

Reply:
Yes boiler plate generation. They have really detailed documentation here: https://book.kubebuilder.io/. It explains quite well all the generated components and how they connect with each other. 
Note: I will be cleaning up some unused files in the  ./config  folder. Mostly kustomize files